docs/developers/omnichain/zrc-20.md

```diff
@@ -94,104 +74,77 @@ contract SystemContract {
-  console.log("Token sent. tx:", tx.hash);
+import { getAddress } from "@zetachain/protocol-contracts";
+
+const main = async (args: any, hre: HardhatRuntimeEnvironment) => {
+  const [signer] = await hre.ethers.getSigners();
+
+  const to = getAddress("tss", hre.network.name as any);
+  const value = parseEther(args.amount);
+
+  const tx = await signer.sendTransaction({ to, value });
+
+  console.log(`Transaction hash: ${tx.hash}`);
 };
-main().catch((error) => {
-  console.error(error);
-  process.exit(1);
-});
+
+task("send", "Send tokens to a TSS address", main).addParam(
+  "amount",
+  "Amount to send to the recipient"
+);
 ```
 
 If you instead wanted to do a `DepositAndCall`, you can do a similar pattern but
 include data in the deposit call. This example demonstrates calling a `swap`
 contract that exists on the zEVM.
 
 ```jsx title="TestDepositAndCall.js"
-import { BigNumber } from "@ethersproject/bignumber";
+import { task } from "hardhat/config";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
 import { parseEther } from "@ethersproject/units";
-import { getAddress, isNetworkName } from "@zetachain/addresses";
-import { ethers } from "hardhat";
-import { ZRC20Addresses, TSS_ATHENS2 } from "../systemConstants";
-import { network } from "hardhat";
-// Helper function to format data for sending a swap transaction
-const getSwapData = (
-  zetaSwap: string,
-  destination: string,
-  destinationToken: string,
-  minOutput: BigNumber
-) => {
-  const params = getSwapParams(destination, destinationToken, minOutput);
-  return zetaSwap + params.slice(2);
-};
-// Primary function definition
-const main = async () => {
-  if (!isNetworkName(network.name) || !network.name)
-    throw new Error("Invalid network name");
-  // Here you're choosing the target token you want the swap to output based on the network
-  const destinationToken =
-    network.name == "goerli"
-      ? ZRC20Addresses["tMATIC"]
-      : ZRC20Addresses["gETH"];
-  console.log("Swapping native token...");
-  // Get a signer to write your transaction
-  const [signer] = await ethers.getSigners();
-  // Get the correct address of the swap contract (using a helper function)
-  const zetaSwap = getAddress({
-    address: "zetaSwap",
-    networkName: "athens",
-    zetaNetwork: "athens",
-  });
-  // Use formatting function to get the correct data format
-  const data = getSwapData(
-    zetaSwap,
-    signer.address,
-    destinationToken,
-    BigNumber.from("0")
+import { getAddress } from "@zetachain/protocol-contracts";
+import { BigNumber } from "@ethersproject/bignumber";
+import { prepareData } from "@zetachain/toolkit/helpers";
+
+const main = async (args: any, hre: HardhatRuntimeEnvironment) => {
+  const [signer] = await hre.ethers.getSigners();
+  console.log(`🔑 Using account: ${signer.address}\n`);
+
+  const network = hre.network.name;
+  const destinationToken = getAddress("zrc20" as any, args.destination as any);
```

Comment:
Any chance to avoid `any`?

Reply:
https://github.com/zeta-chain/docs/pull/71/commits/fc9135a613a59cdacd3eef989ec7cf07ed053ea4